lib/core.js

```diff
@@ -386,6 +386,12 @@ declare class Set<T> {
     values(): Iterator<T>;
 }
 
+declare class WeakSet<T: Object> {
```

Comment:
i think you need to add a constructor, like

```
<V>(iterable: ?Iterable<V>): WeakSet<V>
```

not sure how that works when the param is missing... hopefully we leave it unresolved and the first call to `add` will lock it down. cc @avikchaudhuri